app/controllers/sessions_controller.rb

```diff
@@ -21,4 +21,5 @@ def destroy
     log_out if logged_in?
     redirect_to root_url
   end
+
```

Comment:
Please remove this empty line

Reply:
I know it. I create difference to new commit because I merged to master.